src/events/eventActions.js

```diff
@@ -43,12 +44,16 @@ export const startEventPolling = (queueId: number, eventId: number) => async (
     }
 
     const actions = [];
+    let startTypingNotificationLoop = false;
     for (const event of response.events) {
       lastEventId = Math.max(lastEventId, event.id);
       eventMiddleware(getState(), event);
       const action = eventToAction(getState(), event);
 
       if (action) {
+        startTypingNotificationLoop = !!(
+          action.type === 'EVENT_TYPING_START' && Object.keys(getState().typing).length === 0
```

Comment:
Also, the place for this is in some action creator related to EVENT_TYPING_START. (which we currently do not have)